pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[build-system]
+requires = ["setuptools>=42", "wheel", "scikit-build", "ninja"]
+build-backend = "setuptools.build_meta"
```

Comment:
This line was missed in the update - it's quite important, "setuptools.__legacy__" is the default and often has issues (PyPA-build for example tends to break with the __legacy__ behavior, it also does not catch bugs with the SDist, while the `build_meta` behavior does).